helm/opencensus-collector-app/values.yaml

```diff
@@ -23,3 +23,15 @@ resources:
   requests:
     cpu: 200m
     memory: 512Mi
+
+enableOpencensusReceiver: true
+enableZipkinReceiver: true
+enableJaegerReceiver: true
```

Comment:
What do these do?

Reply:
As part of the exercise: read the changed templates and try to figure out what they do :) If you're lost, we can have a call about this. They are meant to allow the user to configure receivers.

Reply:
thanks! "read the changed templates and try to figure out what they do" yes i tried to do that! For this case, in the original template, I didn't see these parameters (or whatever you call the `enableOpencensusReceiver:` thingies. Got it now! So you can have `enableRandomReceiver: false`

Reply:
Exactly. Application's user can choose this way which receivers should we enable and our chart will include conditionally the necessary config stuff.